docs/release-notes/8.12.asciidoc

```diff
@@ -0,0 +1,75 @@
+* Adds a button to rule execution error messages that lets you ask AI Assistant to diagnose errors ({pull}166778[#166778]).
+
+[discrete]
+[[enhancements-8.12.0]]
+==== Enhancements
+
+* Refactors the timeline UI — various minor updates ({pull}168230[#168230]).
+* Introduces manual saving for Timeline ({pull}171027[#171027], {pull}169239[#169239]).
+* Improves forward-compatibility for the rule schema ({pull}170861[#170861]).
+* Simplifies the format of risk engine API error responses ({pull}170645[#170645]).
+* Makes various UI improvements to the alert details flyout ({pull}170279[#170279], {pull}169035[#169035], {pull}173399[#173399], {pull}170078[#170078], {pull}168297[#168297]).
+* Saves the state of the alert details flyout in the browser. For example, after you use the flyout's *Investigate in timeline* button, you can click your browser's back button to return to the flyout ({pull}169661[#169661]).
+* Integrates a new Event Tracing for Windows (ETW) provider (Microsoft-Windows-Win32k) to create new event types that can be used by prebuilt endpoint rules to detect keylogging activity.
+* Allows for acting and target memory region buffers within behavior alerts to be scanned against {elastic-sec}'s collection of YARA signatures when collected. Detections are added to alerts.
+* Adds a new ReadProcessMemory (lsass) event that can be used by prebuilt endpoint rules to detect credential dumping.
+* Adds a link to the Edit connector UI for Amazon Bedrock that goes to the token tracking dashboard ({pull}172115[#172115]).
+* Allows you to use the `matches` operator when defining Endpoint rule exceptions and trusted applications.
+
+[discrete]
+[[bug-fixes-8.12.0]]
+==== Bug fixes
+
+* Fixes response action bugs by mapping the `unisolate` command to the `release` command and the `running-processes` command to the `processes` command ({pull}173831[#173831]).
+* Fixes the dark theme for the alert details flyout footer ({pull}173577[#173577]).
+* Makes the Timeline tour compatible with the Timeline template page ({pull}173526[#173526]).
+* Stops the {esql} tab from rendering when Timeline isn't open ({pull}173484[#173484]).
+* Fixes a bug that caused {ml} fetch jobs to fail when the default data view (`securitySolution:defaultIndex`) contained special characters ({pull}173426[#173426]).
+* Remove the **Assignees** field from the event details flyout ({pull}173314[#173314]).
+* Fixes a bug that caused the **Add to Case** action to fail if you didn't add a comment before isolating and releasing a host ({pull}172912[#172912]).
+* Fixes a UI bug that overlaid **Default Risk score** values as you created a new rule ({pull}172677[#172677]).
+* Fixes a bug that cleared configured fields in the exceptions flyout after the flyout reloaded and refocused ({pull}172666[#172666]).
+* Fixes the {esql} default query size and appends the query with a `keep` clause ({pull}171789[#171789]).
```

Comment:
As discussed with @PhilippeOberti today, this change will be overwritten (dev PR pending), so I might need to remove this PR.

cc: @michaelolo24

Reply:
Verified with @kqualters-elastic via slack that this can be removed.

```suggestion
```